packages/esm-patient-chart-app/src/visit/visit-form/visit-form.test.tsx

```diff
@@ -67,6 +67,13 @@ describe('VisitForm: ', () => {
   describe('Form submission: ', () => {
     let saveButton: HTMLElement;
 
+    beforeEach(() => {
+      // @ts-ignore
+      jest.useFakeTimers('modern');
+      // @ts-ignore
+      jest.setSystemTime(1638682781000);
```

Comment:
TIL about this Jest functionality, sweet! Where did this number come from though??? Can you add a comment or give it a variable name or something?

Reply:
this is unix timestamp in seconds, updating it with a comment